.travis.yml

```diff
@@ -9,11 +9,11 @@ services:
 
 # Disable automatic submodule fetching ...
 git:
+    depth: 9999999
```

Comment:
Mmmm... I think having such a complete clone just to get a complete changelog is a bit too much, I would just remove the changelog (or let it generate only what we have). Again using makd should help here, otherwise we are duplicating work.